src/main/java/org/elasticsearch/script/ScriptService.java

```diff
@@ -579,6 +609,16 @@ public static void writeTo(ScriptType scriptType, StreamOutput out) throws IOExc
                 out.writeVInt(INLINE_VAL); //Default to inline
             }
         }
+
+        @Override
+        public String toString() {
+            //ScriptModes looks for 'dynamic' rather than 'inline' when reading settings, that's the terminology
+            //we always used with users, although we prefer inline internally
+            if (this == INLINE) {
```

Comment:
been thinking about it... we might want to take the chance to introduce `inline` to users with this PR and remove this conversion. My thinking is that dy `dynamic` up until now we meant indexed scripts + inline scripts (script.disable_dynamic setting would affect both), while now we have a distinction between the two. dynamic in the context of fine-grained settings becomes even more confusing maybe, we should really use inline and indexed now.
Thoughts? Isn't it going to be better for users too? @clintongormley ?